.env

```diff
@@ -1,3 +1,3 @@
-DELIVERY_SDK_VERSION=2.0.4
+DELIVERY_SDK_VERSION=3.0.0
 MANAGEMENT_SDK_VERSION=0.10.1
```

Comment:
shouldn't this be removed also since you did remove all CMA usage

Reply:
why not ;-)